docs/DeveloperGuide.md

```diff
@@ -691,22 +691,81 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Testing features
+
+1. Features with one input
+
+  The following examples are tested on a list containing 1 or more applications and is targeted towards testing the delete command,
+the steps are general enough to be used to test other commands that accept a single parameter. <br>
+   1. Test valid input values
+      - E.G. `delete INDEX` <br>
+        Test input: `delete 1` when list has more than 1 application <br>
+        Expected output: Successful execution of the command
+   2. Test invalid input values
+      - E.G.  `delete INDEX` <br> 
+        Test input: `delete 0` or `delete -10` <br>
+        Expected output: error message detailing what went wrong
+   3. Test invalid command input
+       - E.G.  `delete INDEX` <br>
+       Test input: `delet 1` or `delete 2 2` <br>
+       Expected output:  error message detailing what went wrong
+   
+2. Feature with multiple parameters
+    
+   The following examples are tested on a list containing 1 or more applications and is targeted towards testing the edit command,
+the steps are general enough to be used to test other commands that accept multiple parameters.
+   
+   
+   2. Test valid input values
+      - E.G. `edit INDEX [Optional parameters]` <br>
+        Test input: `edit 1 n/new name` when list has more than 1 application <br>
+        Expected output: Successful change of application 1 name
+   3. Test invalid input values
+      - E.G.  `edit INDEX [Optional parameters]` <br>
+        Test input: `edit 0 n/new name` or `edit -10 n/new name` <br>
+        Expected output: error message detailing what went wrong
+   4. Test invalid command input
+       - E.G.  `edit INDEX [Optional parameters]` <br>
+         Test input: `edi 1 n/new name` or `edit 2 2 n/new name` <br>
+         Expected output: error message detailing what went wrong
+   5. Test multiple parameter combinations
+       - E.G. `edit INDEX [optional parameters]` <br>
+         Test input: `edit 1 n/new name n/actual name` <br>
+         Expected output: error message detailing what went wrong
+         Actual output: command executes
```

Comment:
```suggestion
         Expected output: command executes with the last parameter
```
Is it supposed to have both "Expected output" and "Actual output"?

Reply:
yes, I wrote it from the testers point of view, since typing in multiple would have an unexpected output for our application. We might want to discuss on this, since I wasn't too sure when writing this part